dgfit/old/plot_dustmodel.py

```diff
@@ -45,16 +36,18 @@
     matplotlib.rc("xtick.major", width=2)
     matplotlib.rc("ytick.major", width=2)
 
-    # dustmodel = DustModel(['astro-silicates','astro-graphite'])
-    DM = DustModel()
-    DM.predict_full_grid(
-        ["astro-silicates", "astro-carbonaceous"], path="dgfit/data/indiv_grain/"
+    data_path = pkg_resources.resource_filename("dgfit", "data/")
+    DM = DustModel(
+        ["astro-silicates", "astro-graphite"], path=data_path + "indiv_grain/"
     )
+    # DM = DustModel()
+    # DM.predict_full_grid(b
+    #     ["astro-silicates", "astro-carbonaceous"], path="dgfit/data/indiv_grain/"
+    # )
 
-    if args.obsdata:
-        DM_obs = DustModel()
-        DM_obs.predict_observed_data(DM, OD)
-        DM = DM_obs
+    DM_obs = DustModel()
+    DM_obs.grains_on_obs(DM, OD)
+    DM = DM_obs
```

Comment:
This means one always has to include observed data, right?  Is this needed/appropriate?  Are there cases where we will just want to plot the dust grain model w/o any observed data?  The answer to this question will impact if an error should be raised (see previous comment).